app/views/LivechatEditView.tsx

```diff
@@ -39,20 +42,65 @@ const styles = StyleSheet.create({
 	}
 });
 
-const Title = ({ title, theme }) =>
+interface ITitle {
+	title: string;
+	theme: string;
+}
+
+interface IField {
+	_id: string;
+	visibility: string;
+	scope: string;
+}
+
+interface IInputs {
+	[key: string]: string | string[] | undefined;
+	name: string;
+	email: string;
+	phone?: string;
+	topic: string;
+	tag: string[];
+}
+
+type TParams = IVisitor & IInputs;
+
+interface IInputsRefs {
+	name: RNTextInput | null;
+	phone: RNTextInput | null;
+	topic: RNTextInput | null;
+	[index: string]: RNTextInput | null;
+}
+
+interface ICustomFields {
+	visitor?: { [key: string]: string };
+	livechat?: { [key: string]: string };
+}
+interface ILivechatEditViewProps {
+	user: any;
```

Comment:
Just adding a comment here in case that someone else sees it, but the reason why this is `any` is because it comes from `mapStateToProps`